xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-api/src/main/java/org/xwiki/mentions/internal/MentionFormatterProvider.java

```diff
@@ -20,22 +20,23 @@
 package org.xwiki.mentions.internal;
 
 import org.xwiki.component.annotation.Role;
-import org.xwiki.mentions.DisplayStyle;
+import org.xwiki.mentions.MentionsFormatter;
 
 /**
- * Provides services to format the user mentions.
+ * Provides a {@link MentionsFormatter} component according to the type of actor to format.
  *
  * @version $Id$
- * @since 12.6
+ * @since 12.10RC1
+ * @see MentionsFormatter
  */
 @Role
-public interface MentionsFormatter
+public interface MentionFormatterProvider
```

Comment:
Sounds a bit overkill to have a dedicated role for it: I doubt we'll ever have another implem of it. But it's not harmful IMO.

Reply:
I agree, but I wouldn't bet on the future. I added it mainly for uniformity's sake, but I agree it is a matter of taste.

Reply:
Ok so let's keep it like that.